test/functional/legacy-fixtures/regression/legacy-gh-16/testcafe-fixtures/index.test.js

```diff
@@ -0,0 +1,15 @@
+'@fixture GH-16';
+'@page ./index.html';
+
+
+'@test'['Check stored data'] = {
+    '1.Set data and reload iframe': inIFrame('#iframe', function () {
+        this.data = 200;
+
+        act.click('#replaceSrc');
+    }),
+
+    '2.Check data': function () {
+        eq(this.data, '200');
```

Comment:
Have you checked the test locally? As I see it should be failed since this.data should be equal to `200` but not `'200'`